test/test_model.py

```diff
@@ -65,202 +71,230 @@ def setUp(self):
             secretfoo:
                 type: secret
         ''')
-        self.addCleanup(self.harness.cleanup)
-        self.relation_id_db0 = self.harness.add_relation('db0', 'db')
-        self.harness._get_backend_calls(reset=True)
-        self.model = self.harness.model
+        yield harness
+        harness.cleanup()
+
+    @pytest.fixture
+    def model(self, harness: ops.testing.Harness[ops.CharmBase]):
+        return harness.model
```

Comment:
Yes, we can. I'll remove it then. There are a couple of other places where it can remove a fixture, for example, `container` and `container.pebble`. Shall I go ahead and do it?